mi-scheduler/overlays/test/imagestreamtag.yaml

```diff
@@ -11,3 +12,17 @@ spec:
     importPolicy: {}
     referencePolicy:
       type: Source
```

Comment:
shouldn't this be part of the mi directory inside thoth-application?
i'm creating a PR currently

Reply:
no, other directories are needed. we would keep all the manifest related to mi in one place.
for easy maintainability.

Reply:
as the cronjob mi-scheduler , schedules the mi workflows , so keeping them in one repo, will let argo deploy them in sync.

Reply:
Ok!